src/Adapter/Product/SpecificPrice/CommandHandler/AddProductSpecificPriceHandler.php

```diff
@@ -85,7 +85,6 @@ private function createSpecificPriceFromCommand(AddProductSpecificPriceCommand $
         $specificPrice->reduction_tax = $command->includesTax();
         $specificPrice->price = $command->getPrice();
         $specificPrice->from_quantity = $command->getFromQuantity();
-        $specificPrice->id_shop_group = $command->getShopGroupId() ?? 0;
```

Comment:
So group shop is not used in product's specific prices? (at least not in the edition form)
Why does it exist then? Is it related to catalog price rules maybe?

Reply:
Im not sure, I tend to believe it might be forgotten feature or so...
I don't see it anywhere in UI, nor in productwrapper (which used to do all the work https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Product/AdminProductWrapper.php#L281) and I didn't spot any meaningful usages anywhere else :man_shrugging:

Reply:
The important is to check if it has impact in the FO, but looking at `SpecificPrice::getSpecificPrice` I see no reference to the group So I think it's ok to remove it indeed An you can also clean it in `SpecificPriceForEditing` and `AddProductSpecificPriceCommand`